google/src/main/scala/org/broadinstitute/dsde/rawls/metrics/GoogleInstrumentedService.scala

```diff
@@ -0,0 +1,14 @@
+package org.broadinstitute.dsde.rawls.metrics
+
+/**
+  * An instrumented Google service.
+  */
+object GoogleInstrumentedService extends Enumeration {
+  type GoogleInstrumentedService = Value
+  val Billing, Buckets, Genomics, Groups, PubSub = Value
```

Comment:
Storage instead of Buckets makes sense. I think I didn't reuse `Subsystem` here because workbenchGoogle doesn't depend on rawlsModel. (Which reminds me - I still need to port these changes to https://github.com/broadinstitute/workbench-libs).